### PR TITLE
Remember window position on screen

### LINKED ISSUE #397: Remember window position on screen
Could you add an option to let PDF arranger remember its window position (not only size) on screen at exit.

### PR DESCRIPTION
**Pull request by JATIN2111999**
hey i fixed this issue you can check the patch

### CODE REVIEW

File: pdfarranger/config.py [hunk 1]
```diff
@@ -111,6 +111,13 @@ def set_window_size(self, size):
         self.data.set('window', 'width', str(size[0]))
         self.data.set('window', 'height', str(size[1]))
 
+    def set_position(self,position):
+        self.data.set('window','root_x',str(position[0]))
+        self.data.set('window','root_y',str(position[1]))
```
Comment (dreua) [inline pdfarranger/config.py:116]: Please add a space after each comma here

File: pdfarranger/config.py [hunk 2]
```diff
@@ -111,6 +111,13 @@ def set_window_size(self, size):
...
+
+    def position(self):
+        return self.data.getint('window', 'root_x', fallback=10),self.data.getint('window', 'root_y', fallback=10)
```
Comment (dreua) [inline pdfarranger/config.py:119]: Space missing, too

**Review by dreua — CHANGES_REQUESTED** [review]
Hey, thanks for the PR! Code looks good to me except for some missing spaces.
Could you remove the "fixes #number:" from the commit title and add it as `Closes: https://github.com/pdfarranger/pdfarranger/issues/397` in the commit message body instead? Squashing everything into one commit would also be appreciated but I can do this through Github, too.

File: pdfarranger/config.py (hunk 1 above)
Comment (dreua) [reply pdfarranger/config.py:116]: Not sure why the CI doesn't notice this, but I'm quite sure that is the style we use throughout the project.

Comment (JATIN2111999) [reply pdfarranger/config.py:116]: got it

File: pdfarranger/config.py (hunk 2 above)
Comment (JATIN2111999) [reply pdfarranger/config.py]: done

File: pdfarranger/config.py (hunk 1 above)
Comment (JATIN2111999) [reply pdfarranger/config.py]: done

**Comment by JATIN2111999** [comment]
Hey i did the changes if you have any issue related to PR let me know :)

**Comment by JATIN2111999** [comment]
> Hey, thanks for the PR! Code looks good to me except for some missing spaces.
> Could you remove the "fixes #number:" from the commit title and add it as `Closes: https://github.com/pdfarranger/pdfarranger/issues/397` in the commit message body instead? Squashing everything into one commit would also be appreciated but I can do this through Github, too.

done

**Comment by dreua** [comment]
Superseded by #450

**Pull request closed without merging**